src/components/Tabs/README.md

```diff
@@ -25,33 +184,131 @@
+                )} size={16} />
+            }
+            id="first"
+            title="Tab with icon"
+            onClick={() => {}}
+        />
+        <UIKit.Tabs.Item id="second" title="Tab without icon" onClick={() => {}} />
+    </UIKit.Tabs>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Tabs activeTab="first">
+  <Tabs.Item
+    icon={<Icon size={16} data={GearIcon} />}
+    id="first"
+    title="Tab with icon"
+    onClick={() => {}}
+  />
+  <Tabs.Item id="second" title="Tab without icon" onClick={() => {}} />
+</Tabs>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### States
+
+Tabs item has disabled flag.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Tabs activeTab="first">
+    <Tabs.Item id="first" title="First Tab" onClick={() => {}} />
+    <Tabs.Item disabled id="second" title="Disabled Tab" onClick={() => {}} />
+</Tabs>
+`}
+>
+    <UIKit.Tabs activeTab="first">
+        <UIKit.Tabs.Item id="first" title="First Tab" onClick={() => {}} />
+        <UIKit.Tabs.Item disabled id="second" title="Disabled Tab" onClick={() => {}} />
+    </UIKit.Tabs>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Tabs activeTab="first">
+  <Tabs.Item id="first" title="First Tab" onClick={() => {}} />
+  <Tabs.Item disabled id="second" title="Disabled Tab" onClick={() => {}} />
+</Tabs>
 ```
+
+<!--/GITHUB_BLOCK-->
+
+### Counter
+
+Used if you need to display a number for a tabs item.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Tabs activeTab="first">
+    <Tabs.Item id="first" title="First Tab" onClick={() => {}} counter={13} />
+    <Tabs.Item id="second" title="Second Tab" onClick={() => {}} counter={3} />
+</Tabs>
+`}
+>
+    <UIKit.Tabs activeTab="first">
+        <UIKit.Tabs.Item id="first" title="First Tab" onClick={() => {}} counter={13} />
+        <UIKit.Tabs.Item id="second" title="Second Tab" onClick={() => {}} counter={3} />
+    </UIKit.Tabs>
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Tabs activeTab="first">
+  <Tabs.Item id="first" title="First Tab" onClick={() => {}} counter={13} />
+  <Tabs.Item id="second" title="Second Tab" onClick={() => {}} counter={3} />
+</Tabs>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Properties
+
+| Name     | Type                        | Required | Default | Description                      |
```

Comment:
Order of columns is: Name, Description, Type, Default. Type and Default are centered